WordPress/Classes/ViewRelated/Blog/Site Monitoring/WebServerLogsView.swift

```diff
@@ -170,13 +175,33 @@ private struct WebServerLogsRowView: View {
                     .font(.system(.footnote))
                     .foregroundStyle(.secondary)
             }
-            Text(entry.requestUrl ?? "")
-                .font(.system(.subheadline))
-                .lineLimit(3)
+            WebServerLogUrlLabel(text: entry.requestUrl ?? "", width: width)
         }
     }
 }
 
+private struct WebServerLogUrlLabel: UIViewRepresentable {
+    let text: String
+    let width: CGFloat
+
+    func makeUIView(context: Context) -> UILabel {
+        let label = UILabel()
+        label.text = text
+        label.preferredMaxLayoutWidth = width
```

Comment:
I doubted if it was needed, but based on how UIKit-SwiftUI layout system integration works, looks like it is 😞